app/institutions/dashboard/-components/object-list/template.hbs

```diff
@@ -0,0 +1,57 @@
+<IndexCardSearcher
+    @queryOptions={{@defaultQueryOptions}}
+    @sort={{this.sort}}
+    @page={{this.page}}
+as |list|>
+    <Institutions::Dashboard::-Components::InstitutionalDashboardWrapper @institution={{@institution}} as |wrapper|>
+        <wrapper.main local-class='main-column'>
+            {{#if list.searchObjectsTask.isRunning}}
+                <LoadingIndicator @dark={{true}} />
+            {{else}}
+                <table data-test-object-list-table local-class='object-table'>
+                    <thead local-class='object-table-head'>
+                        <tr>
+                            {{#each @columns as |column|}}
+                                <th local-class='object-table-header'>
+                                    {{column.name}}
+                                </th>
+                            {{/each}}
+                        </tr>
+                    </thead>
+
+                    <tbody>
+                        {{#each list.searchResults as |result|}}
+                            <tr>
+                                {{#each @columns as |column|}}
+                                    <td>
+                                        {{#if (eq column.type 'link')}}
+                                            <Institutions::Dashboard::-Components::ObjectList::LinkField
+                                                @searchResult={{result}}
+                                                @getHref={{column.getHref}}
+                                                @getLinkText={{column.getLinkText}}
+                                            />
+                                        {{else if (eq column.type 'doi')}}
+                                            <Institutions::Dashboard::-Components::ObjectList::DoiField
+                                                @searchResult={{result}}
+                                            />
+                                        {{else if (eq column.type 'contributors')}}
+                                            <Institutions::Dashboard::-Components::ObjectList::ContributorsField
+                                                @searchResult={{result}}
+                                                @institution={{@institution}}
+                                            />
+                                        {{else}}
+                                            <Institutions::Dashboard::-Components::ObjectList::TextField
+                                                @searchResult={{result}}
+                                                @getterFunction={{column.getValue}}
+                                            />
```

Comment:
i'll admit i thought ember-osf-web was on a version of ember that could use [plain functions as helpers](https://guides.emberjs.com/release/components/helper-functions/#toc_local-helper-functions) (so this could be simply `{{column.getValue result}}`) but alas, not even close

...tho it does have `ember-composable-helpers`, which provides a `call` helper -- this could be `{{call column.getValue result}}`

Reply:
Whoa, I was not aware of `{{call}}`! Seems like exactly the thing I was looking for!